meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -312,7 +312,7 @@ board_config() {
               BIOS_HASH_LINK_DES="${BIOS_LINK_DES}.sha256"
               BIOS_SIGN_LINK_COMM="${BIOS_LINK_COMM}.sha256.sig"
               BIOS_SIGN_LINK_DES="${BIOS_LINK_DES}.sha256.sig"
-              PLATFORM_SIGN_KEY="dasharo/dasharo-release-1.x-compatible-with-msi-ms-7d25-signing-key.asc"
+              PLATFORM_SIGN_KEY="dasharo/msi_ms7d25/dasharo-release-1.x-compatible-with-msi-ms-7d25-signing-key.asc"
```

Comment:
Shouldn't we be using the variable DASHARO_REL_NAME (msi_ms7d25) instead of hard-coding the path?

Reply:
@Firminator, very good point. @artur-rs I would add this to a roadmap for the next release.